src/Kernel-Tests/MyMessageNotUndestoodDummy.class.st

```diff
@@ -0,0 +1,17 @@
+"
+For the Class part:  ""I represent a MessageNotUnderstood with overriden methods for unit tests"".
+
+For the Responsibility part: Used for testing the base class.
+
+"
+Class {
+	#name : #MyMessageNotUndestoodDummy,
+	#superclass : #MessageNotUnderstood,
+	#category : #'Kernel-Tests-Exceptions'
+}
+
+{ #category : #accessing }
+MyMessageNotUndestoodDummy >> messageBasedDescription [ 
+	"Overrides the method for testing"	
+	^ 'messageBasedDescription'
+]
```

Comment:
I would prefer that we directly instantiate and fill a real `MessageNotUnderstood` to test it rather than creating a dummy class. It's fine to do so but here I think we can easily create a MNU object and work with it to test its description behavior.

Reply:
I tried to instantiate. However, the reason for the dummy class is that there are some required fields, for the production of the description, that I am not capable of manually creating them. Overriding that method was a quick fix.
For this class "MessageNotUnderstood" I used a real MessageNotUnderstood exception for the test in the latest push.